DependencyInjection/ApacheTikaExtension.php

```diff
@@ -25,29 +25,23 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $container->setParameter('apache_tika.tika_path', isset($config['tika_path']) ? $config['tika_path'] : null);
-        $container->setParameter('apache_tika.tika_host', isset($config['tika_host']) ? $config['tika_host'] : null);
-        $container->setParameter('apache_tika.tika_port', isset($config['tika_port']) ? $config['tika_port'] : null);
+        $path = isset($config['path']) ? $config['path'] : null;
```

Comment:
Note sure it makes sense since this value is the one used to trigger whether or not the user want the cli client